tests/mask.py

```diff
@@ -37,7 +37,7 @@
 
 import sys
 import unittest
-import numpy
+import numpy as np
```

Comment:
At risk of sounding too pedantic, I suggest making no-op cleanups like this (and cleaning up whitespace and indentation) as a separate commit from changes that change the behavior. In this case it may not be worth going back (though it's certainly doable).